packages/plugins/RedPacket/src/SiteAdaptor/FIreflyRedPacketInSentHistoryList.tsx

```diff
@@ -0,0 +1,263 @@
+import { TokenIcon } from '@masknet/shared'
+import { NetworkPluginID } from '@masknet/shared-base'
+import { makeStyles } from '@masknet/theme'
+import { useNetworkDescriptor } from '@masknet/web3-hooks-base'
+import {
+  FireflyRedPacketAPI,
+} from '@masknet/web3-providers/types'
+import { formatBalance } from '@masknet/web3-shared-base'
+import { Box, ListItem, Typography } from '@mui/material'
+import { memo } from 'react'
+import { RedPacketTrans, useRedPacketTrans } from '../locales/index.js'
+import { dateTimeFormat } from './utils/formatDate.js'
+import { FireflyRedPacketActionButton } from './FireflyRedPacketActionButton.js'
+
+const useStyles = makeStyles<{ listItemBackground?: string; listItemBackgroundIcon?: string }>()((
+  theme,
+  { listItemBackground, listItemBackgroundIcon },
+) => {
+  const smallQuery = `@media (max-width: ${theme.breakpoints.values.sm}px)`
+  return {
+    message: {
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      [smallQuery]: {
```

Comment:
Looks like you bypass the git commit hook, code indent width should be 4 spaces instead of 2 ones. Follow our code style, you can format with prettier.